docs/source/create/avatars/mixamo-tutorial.md

```diff
@@ -7,23 +7,21 @@
 
```

Comment:
As this tutorial depends on Adobe Fuse, we should update the warning at the top to indicate that the tutorial will need to be revisited.